2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_setup.py:_flush():67] Configure stats pid to 691275
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_setup.py:_flush():67] Loading settings from /home/pallette/.config/wandb/settings
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_setup.py:_flush():67] Loading settings from /home/pallette/Documents/GitHub/Bachelor/wandb/settings
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/pallette/Documents/GitHub/Bachelor/wandb/run-20250316_132910-azkkm2vs/logs/debug.log
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/pallette/Documents/GitHub/Bachelor/wandb/run-20250316_132910-azkkm2vs/logs/debug-internal.log
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_init.py:init():761] calling init triggers
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'model': 'KeyNet', 'dataset': 'cropped_objects', 'batch_size': 24, 'val_batch_size': 24, 'learning_rate': 0.001, 'global_image_size': (700, 700), 'num_epochs': 20, 'num_channels': 3, 'gaussian_blur': True, 'post_processing_threshold': 0.5, 'distance_threshold': 10, 'feature_extractor_lvl_amount': 5, '_wandb': {}}
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_init.py:init():784] starting backend
2025-03-16 13:29:10,237 INFO    MainThread:691275 [wandb_init.py:init():788] sending inform_init request
2025-03-16 13:29:10,243 INFO    MainThread:691275 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-16 13:29:10,243 INFO    MainThread:691275 [wandb_init.py:init():803] backend started and connected
2025-03-16 13:29:10,244 INFO    MainThread:691275 [wandb_init.py:init():896] updated telemetry
2025-03-16 13:29:10,251 INFO    MainThread:691275 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-16 13:29:10,783 INFO    MainThread:691275 [wandb_init.py:init():995] starting run threads in backend
2025-03-16 13:29:10,864 INFO    MainThread:691275 [wandb_run.py:_console_start():2377] atexit reg
2025-03-16 13:29:10,864 INFO    MainThread:691275 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-16 13:29:10,865 INFO    MainThread:691275 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-16 13:29:10,865 INFO    MainThread:691275 [wandb_run.py:_redirect():2317] Redirects installed.
2025-03-16 13:29:10,866 INFO    MainThread:691275 [wandb_init.py:init():1037] run started, returning control to user process
2025-03-16 14:49:14,016 WARNING MsgRouterThr:691275 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-03-16 14:49:14,018 INFO    MsgRouterThr:691275 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
